runtime/service_integ_test.go

```diff
@@ -265,7 +264,9 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 	// container ends up in the right VM by assigning each VM a network device with a unique mac address and having each container
 	// print the mac address it sees inside its VM.
 	var vmWg sync.WaitGroup
-	for vmID, c := range cases {
+	for i := 0; i < numberOfVms; i++ {
+		caseTypeNumber := 1 % len(cases)
```

Comment:
This needs to be `i % len(cases)`. I had a typo in my earlier comment!

Reply:
Oh, yeah;)  fixed.